packages/cli/src/utils/getQuestions.ts

```diff
@@ -1,38 +1,49 @@
 import chalk from "chalk";
 import { QuestionCollection } from "inquirer";
 
-export interface GetQuestionsInput {
+import { AgenticaStarter } from "../functional/AgenticaStarter";
+import { IAgenticaStart } from "../structures/IAgenticaStart";
+
+export interface GetQuestionsProps {
   services: {
     name: string;
     value: string;
   }[];
+
+  options: IAgenticaStart.IOptions;
 }
 
+/**
+ * Get questions for `start` command.
+ */
 export const getQuestions = (
-  input: GetQuestionsInput,
+  input: GetQuestionsProps,
 ): QuestionCollection[] => {
-  return [
+  const questions = [
     {
       type: "list",
       name: "packageManager",
       message: "Package Manager",
       choices: [
         "npm",
         "pnpm",
-        `yarn (berry ${chalk.blueBright("is not supported")})`,
-      ],
-    },
-    {
-      type: "list",
-      name: "projectType",
-      message: "Project Type",
-      choices: [
-        `NodeJS ${chalk.blueBright("Agent Server")}`,
-        `NestJS ${chalk.blueBright("Agent Server")}`,
-        `React ${chalk.blueBright("Client Application")}`,
-        `Standard ${chalk.blueBright("Application")}`,
+        {
+          name: `yarn (berry ${chalk.blueBright("is not supported")})`,
+          value: "yarn",
+        },
       ],
     },
+    input.options.project
+      ? null
+      : {
+          type: "list",
+          name: "projectType",
+          message: "Project Type",
+          choices: Object.values(AgenticaStarter.PROJECT).map((project) => ({
+            name: project.title,
+            value: project.key,
```

Comment:
I think this part looks better before than now.
It's just NIT. Don't think carefully.

Reply:
It looked prettier before,
but I made the changes below to reduce maintenance points.

Reply:
got you